spec/controllers/v0/facilities/cemetery_controller_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe V0::Facilities::CemeteryController, type: :controller do
```

Comment:
What do these controller specs provide you that cannot be tested with request specs?